0_portfolio_input_check_functions.R

```diff
@@ -244,7 +243,7 @@ map_security_sectors <- function(fin_data, sector_bridge) {
   fin_data %>%
     group_by(security_mapped_sector) %>%
     filter(is.na(security_mapped_sector)) %>%
-    summarise(count = n())
+    summarise(count = n(), .groups = "drop")
```

Comment:
I see

```r
Quitting from lines 251-264 (integration-test.Rmd) 
Error: [conflicted] `summarize` found in 2 packages.
Either pick the one you want with `::` 
* plyr::summarize
* dplyr::summarize
Or declare a preference with `conflict_prefer()`
* conflict_prefer("summarize", "plyr")
* conflict_prefer("summarize", "dplyr")
```

but this error should go away after we merge https://github.com/2DegreesInvesting/PACTA_analysis/pull/179

Reply:
Actually, this is a different conflict. Do we need plyr at all? It has been superseded by dplyr for a long time.